src/helpers/dimensions.ts

```diff
@@ -255,6 +255,28 @@ export function getHeadlinesDimensions(): AFM.IDimension[] {
     return [{ itemIdentifiers: ["measureGroup"] }];
 }
 
+export function getXirrDimensions(
+    mdObject: VisualizationObject.IVisualizationObjectContent,
+): AFM.IDimension[] {
+    const attribute: VisualizationObject.IBucket = mdObject.buckets.find(
+        bucket => bucket.localIdentifier === ATTRIBUTE,
+    );
+
```

Comment:
```
const itemIdentifiers: attribute && attribute.items.length
? [MEASUREGROUP, ...attribute.items.map(getLocalIdentifierFromAttribute)]
: [MEASUREGROUP];
return [
        {
            itemIdentifiers,
        },
];
```

Reply:
I opted for a pattern that is used all over the file (I would prefer to use the ternary, but I did not want to change the rest of the file).